### ISSUE TITLE
add changelog

### ISSUE THREAD

**Issue by mathpluscode** [DeepRegNet/DeepReg#462]
## Subject of the documentation

Since the release of `0.1.0b1`, we have fixed a few bugs and added a few features.

It would be nice to have a changelog. For now, we can do it manually, 
and each PR into the main should update the changelog so that everything is tracked ;)

**Comment by mathpluscode** [comment]
# Changelog 0.1.0b2

Added (for new features):
- Added visualization tool for generating gifs from model outputs.
- Added `log_root` argument for training and prediction to customize the log file location.
- Added `environment.yml` file for Conda environment creation.
- Added Dockerfile.
- Added documentation about using UCL cluster with DeepReg.

Changed/Updated (for existing functionalities):
- Updated instruction on Conda environment creation.
- Updated TensorFlow version to 2.3.1.
- Updated the issue and pull-request templates to fix some 
- Updated the pre-trained models in MR brain demo.
- Updated pre-commit hooks version.
- Updated references in JOSS paper to address reviewers' comments.

Deprecated (for soon-to-be removed features):
- None.

Removed (for removed features):
- Removed integration tests on training and prediction.

Fixed (for bugs): 
- Fixed tensor comparison in unit tests and impacted tests.
- Removed normalization of DDF/DVF when saving in Nifti formats.
- Fixed invalid link in the quick start page.

**Comment by mathpluscode** [comment]
@zacbaum @s-sd @sbb-gh @zacbaum @acasamitjana @RemiDelaunay I've included our changes as above since the first release (acc to the PRs), the categories come from https://keepachangelog.com/en/1.0.0/.

I've tagged you as you have contributed to the package since then, did I miss anything?

@ebonmati @NMontanaBrown What do you think about this kind of changelog ;)

**Comment by ebonmati** [comment]
Looks good to me
